src/monodroid/jni/cxx-abi/bionic_futex.hh

```diff
@@ -0,0 +1,84 @@
+/*
```

Comment:
*This* file should likely be in `src-ThirdParty/ndk`, or some other "meaningful" `src-ThirdParty` subdirectory.